testing/regress/hpcc/util/ecl/command.py

```diff
@@ -157,7 +157,7 @@ def runCmd(self, cmd, cluster, eclfile, report, **kwargs):
                     test = True
                 elif (res['state'] == 'failed') and (('error' in data) or ('exception' in data.lower())):
                     eclfile.diff = ("%3d. Test: %s\n") % (eclfile.taskId, eclfile.getBaseEclRealName())
-                    eclfile.diff += data
+                    eclfile.diff += repr(data).replace("\\xe2\\x80\\x98", "'").replace("\\xe2\\x80\\x99", "'")
```

Comment:
Would a more general "Convert to ascii ignoring any errors" call here be more appropriate?